### PR TITLE
chore: Add documentation for custom strategies

### LINKED ISSUE #338: How to do scheduled feature toggle?
Are we able to do something like: at 10pm on 31st Jul, turn on this feature toggle?

Use case: maintenance window

### PR DESCRIPTION
**Pull request by ivarconr**
Closes #338 
Related #355

### CODE REVIEW

**Comment by ivarconr** [comment]
/cc @VictorJSV what you think? 

Preview:
https://deploy-preview-363--stoic-agnesi-7db75d.netlify.com/docs/activation_strategy_custom

File: docs/acitvation-strategies-custom.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+After we have have created the strategy definition, we can now decide to use that activation strategy for our feature toggle.
+
+![timestamp_use_strategy](assets/timestamp_use_strategy.png)
+
+In the example we want to use our custom strategy for the feature toggle named `demo.TimeStampRollout`.
+
+#### Client implemenation
+
+For the clients to understand the new custom strategy we need to implement the logic to understand the new custom strategy. All official client SDK's for Unleash provides abstractions for you to implement your own custom strategies.
```
Comment (VictorJSV) [inline docs/acitvation-strategies-custom.md]: > the new custom strategy we need to implement the logic to understand the new custom strategy.

this is redundancy

File: docs/acitvation-strategies-custom.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+#### Use custom strategy
+
+After we have have created the strategy definition, we can now decide to use that activation strategy for our feature toggle.
+
+![timestamp_use_strategy](assets/timestamp_use_strategy.png)
+
+In the example we want to use our custom strategy for the feature toggle named `demo.TimeStampRollout`.
+
+#### Client implemenation
```
Comment (VictorJSV) [inline docs/acitvation-strategies-custom.md]: `implemenation` to `implementation`

File: docs/acitvation-strategies-custom.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+In Node.js the implementation for the `TimeStampStrategy` would be:
+
+```javascript
+class TimeStampStrategy extends Strategy {
+  constructor() {
+    super('TimeStamp');
+  }
+
+  isEnabled(parameters, context) {
+    const enableAfterTimeStamp = moment(parameters.enableAfter);
```
Comment (VictorJSV) [inline docs/acitvation-strategies-custom.md]: it could be explain, why use `moment` package.

File: website/sidebars.json [hunk 4]
```diff
@@ -13,7 +13,7 @@
       "database_schema",
       "database_backup"
     ],
-    "Guides": ["guides/google_auth"]
+    "Guides": ["guides/google_auth", "activation_strategy_custom"]
```
Comment (VictorJSV) [inline website/sidebars.json]: it could be in `guides/` folder

File: docs/guides/custom-acitvation-strategy.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+const { Strategy, initialize, isEnabled } = require('unleash-client');
+
+class TimeStampStrategy extends Strategy {
+  constructor() {
+    super('TimeStamp');
+  }
+
+  isEnabled(parameters, context) {
+    const enableAfterTimeStamp = moment(parameters.enableAfter);
+    return moment().isAfter(enableAfterTimeStamp);
```
Comment (SimenB) [inline docs/guides/custom-acitvation-strategy.md]: `Date.parse(parameters.enableAfter) > Date.now()`?

Comment (ivarconr) [reply docs/guides/custom-acitvation-strategy.md]: Did not even consider to use raw dates. Will probably work too!

Comment (ivarconr) [reply docs/guides/custom-acitvation-strategy.md]: I think you have it the wrong way though. Should be:
`Date.parse(parameters.enableAfter) < Date.now()`

Comment (ivarconr) [reply docs/guides/custom-acitvation-strategy.md]: https://github.com/Unleash/unleash/commit/05de035163944f08a0196dd231a81f8df0646f86

**Comment by VictorJSV** [comment]
The name's file has a mistake `custom-acitvation-strategy.md`

Should be `activation`

**Comment by VictorJSV** [comment]
keep calm! take a coffee xD

**Pull request merged**